### DISCUSSION TITLE
Completely lost regarding Salt

### DISCUSSION THREAD

**Discussion by ANDERSONTECHWERKS** [Security-Onion-Solutions/securityonion#3519] [Unsupported Versions]
Just installed SO2 on a VM, and I'm having trouble doing some relatively simple stuff - such as bringing all the traffic from my monitor interface into Alerts / Suricata. 

If someone could point me a direction in regards to how to assign interfaces for each (Which I'm assuming is all done by Salt, but I'm uncertain) that would be great.

My setup is fairly simple. Have an SO instance with access to separate physical management / monitor interfaces. Have confirmed via tshark that my monitor interface is seeing all the traffic I want to monitor, but I'm unable to get any alerts in the SOC. Nor do I see any related information in Kibana.

Any help is appreciated.

**Comment by weslambert** [comment]
Security Onion setup configures a logical `bond0` interface that aggregates all chosen monitoring interfaces.  Have you tried tcpdumping on `bond0` to ensure you are seeing the traffic you expect?
